docs/parcels.md

```diff
@@ -0,0 +1,110 @@
+# Parcels
+
+A single-spa parcel is a bundle of functionality (like an application or component) meant to be controlled imperatively 
+by an application, parcels use similar methodology as applications, and can be considered an advanced single spa feature so we recommend [starting with applications.](/docs/applications.md)
+A parcel can be as large as an application or as small as an component and written in 
+any language as long as it exports the correct lifecycle events. In a single-spa world, your SPA contains 
+many registered applications and potentially many parcels. Typically we recommend you mount a parcel within 
+the context of an application because the parcel will be unmounted with the application.
+
+## Parcel Lifecycles
+Start with [applications](/docs/applications.md#registered-application-lifecycle) to learn more about the functionality of single-spa's lifecycle methods.
+
+### Bootstrap
+This lifecycle function will be called once, right before the parcel is
+mounted for the first time.
+
+```js
+function bootstrap(props) {
+  return Promise
+    .resolve()
+    .then(() => {
+        // This is where you do one-time initialization
+        console.log('bootstrapped!')
+        });
+}
+```
+
+### Mount
+If the parcel is not mounted this lifecycle function is called when ever `mountParcel` is called. When 
+called, this function should create DOM elements, DOM event listeners, etc. to render content to the user.
+
+```js
+function mount(props) {
+  return Promise
+    .resolve()
+    .then(() => {
+        // This is where you tell a framework (e.g., React) to render some ui to the dom
+        console.log('mounted!')
+        });
+}
+```
+
+### Unmount
+This lifecycle function will be called whenever the parcel is mounted and one of the following cases is true:
+- `unmount()` is called
+- The parent parcel or application is unmounted
+When called, this function should clean up all DOM elements, DOM event listeners, leaked memory, globals, 
+observable subscriptions, etc. that were created at any point when the parcel was mounted.
+
+```js
+function unmount(props) {
+  return Promise
+    .resolve()
+    .then(() => {
+        // This is where you tell a framework (e.g., React) to unrender some ui from the dom
+        console.log('unmounted!');
+        });
+}
+```
+
+### Update (optional)
+This lifecycle function will be called whenever the parcel is mounted, and it needs to be updated.
+
+## Example use cases
+
+### Modals
```

Comment:
I like this example -- I think it explains a good, practical scenario where parcels are useful.